Initial review of the Umbraco Search docs (#7983)
umbraco/UmbracoDocs: 17/umbraco-search/getting-started/built-in-property-editors.md

The following list shows how the [built-in Umbraco property editors](../../umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors) are indexed for Umbraco Search.

> Some property editors are excluded because they generate more noise than value in the index. These include the color picker and media pickers.

Reviewer: Could potentially be more ... so maybe:

```suggestion
Some property editors are excluded because they generate more noise than value in the index. For example, color and media pickers have been omitted.
```

Author: Thanks @kjac !
I've updated the article directly via GitBook 🙏

> Some property editors are deliberately excluded because they generate more noise than value in the index. These include color pickers and media pickers.

Reviewer 2: ```suggestion
Some property editors are excluded because they generate more noise than value in the index. These include the color picker and media pickers.
```
